src/sheet/types.ts

```diff
@@ -52,3 +52,9 @@ export interface SheetState {
   sheetId: string;
   rows: SheetRow[]; // key being column id
 }
+
+export enum SheetViewMode {
```

Comment:
Hmm, we don;t use enum in other places - like e.g. look how ButtonVariant is defined

Reply:
https://stackoverflow.com/questions/40275832/typescript-has-unions-so-are-enums-redundant